scripts/pylib/pytest-twister-harness/src/twister_harness/helpers/shell.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import logging
+import re
+import time
+
+from twister_harness.device.device_adapter import DeviceAdapter
+from twister_harness.exceptions import TwisterHarnessTimeoutException
+
+logger = logging.getLogger(__name__)
+
+
+class Shell:
```

Comment:
```suggestion
class Shell:
    prompt: str = 'uart:~$'
    base_timeout: float = 60.0

    def __init__(self, device: DeviceAdapter) -> None:
        self.device: DeviceAdapter = device
```
jeśli będzie potrzeba, użytkownik może zmienic prompta i/lub timeout wywołąniem bezpośrednim.
I chyba nie ma potrzeby "ukrywać" zmiennych dodając podkreślenie